static/js/iframe-staging.js

```diff
@@ -1,15 +1,17 @@
 import { generateIFrame, sendToIframe } from './iframe-common';
 import InjectedData from './models/InjectedData';
 import generateInitAnswersExperienceFrameFunction from './utils/generateInitAnswersExperienceFrameFunction';
-import { RuntimeConfig } from './runtime-config';
+import RuntimeConfig from './runtime-config';
 
-const runtimeConfig = new RuntimeConfig();
-window.setRuntimeConfig = (key, value) => {
-  runtimeConfig.set(key, value);
-  sendToIframe({ runtimeConfig: runtimeConfig.getObject() });
-}
+const answersDataAttributes = document.querySelector('#answers-container')?.dataset;
+const runtimeConfig = new RuntimeConfig(answersDataAttributes);
+runtimeConfig._onUpdate(updatedConfig => {
+  sendToIframe({ runtimeConfig: updatedConfig });
+});
+window.AnswersExperienceFrame = {
+  runtimeConfig: runtimeConfig,
+  init: generateInitAnswersExperienceFrameFunction(runtimeConfig)
+};
```

Comment:
This logic block is the same between iframe-staging and iframe-prod. Could we refactor it such that it lives in iframe-common?

Reply:
what do you think about making AnswersExperienceFrame a simple class?

Reply:
Updated to put this logic into AnswersExperienceFrame